sprint-guide/exercises/schedule-interviews.md

```diff
@@ -1,9 +1,9 @@
 ---
-tags: ['exercise', 'planning']
+tags: ["exercise", "planning"]
 title: Schedule 4–6 people for testing interviews
 ---
 
-During the [Test phase](/phases/test), we'll be running user interviews and
+During the [Test phase](/sprint-guide/phases/test), we'll be running user interviews and
```

Comment:
Same as [above.](https://github.com/thoughtbot/design-sprint-guide/pull/113/files#r1571516395)

Reply:
This one is currently broken, so will keep this change.